.github/workflows/build-and-push-soci-index.yml

```
@@ -15,6 +15,16 @@ on:
         description: 'Name of the image artifact, usually the repository name e.g mlfoundry-server'
         required: true
         type: string
+      enable_ecr:
```

```suggestion
      enable_public_ecr:
```

or take the ecr url if we also want to support private